eclair-core/src/main/scala/fr/acinq/eclair/blockchain/fee/BitgoFeeProvider.scala

```diff
@@ -45,6 +45,8 @@ class BitgoFeeProvider(chainHash: ByteVector32, readTimeOut: Duration)(implicit
         .send()
       feeRanges = parseFeeRanges(res.unsafeBody)
     } yield extractFeerates(feeRanges)
+
+  override def getName: String = "Bitgo"
```

Comment:
I find it weird that the implementation of one particular provider has impacts on other providers. I'd rather remove the `providerName` from the db.

If one day we want to combine multiple providers into one `MultiProvider` (say we want to do averages or something), then this new provider will probably have to use that kind of things internally. But it's a different concern, and even in that case other providers would be left untouched.

Reply:
The feerates is used throughout the app and is an important piece of data, and with all the inheritance going on with providers we don't know where it's taken from. Having a name fixes that.

However by adding a `getName` method, all providers must then implement it including those that do not actually get a feerates but only add some intelligence to it, like `SmoothFeeProvider` or this `DbFeeProvider`, where the name is irrelevant. It could be improved by reworking the design of the providers but it seems out of the scope of this PR. Or maybe there's a smarter way.

Anyway I think it should be done in another PR, we can remove the provider name from the db and from the providers.